src/components/NotificationError/NotificationError.tsx

```diff
@@ -0,0 +1,43 @@
+import {
+  FC,
+  useEffect,
+} from 'react';
+
+import cn from 'classnames';
+
+type Props = {
+  error: string;
+  resetError: () => void;
+};
+
+const NotificationError: FC<Props> = ({ error, resetError }) => {
+  useEffect(() => {
```

Comment:
You should add `resetError` to the useEffect dependency array to avoid potential issues.